packages/headless/src/ssr.index.ts

```diff
@@ -6,17 +6,26 @@ polyfillCryptoNode();
 export type {Unsubscribe, Middleware} from '@reduxjs/toolkit';
 
 // ⚠️ NOTE: All exported SSR types, APIs should be marked as `@internal` until MVP is complete
+export {mapObject} from './utils/utils';
```

Comment:
I would prefer not to export generic utilities like `mapObject` from the library. I would be ok copy and pasting it into the new library.

Reply:
Makes sense. Added a TODO to use a third-party lib if possible and if it doesn't increase bundle size - on one hand it is just handful of lines of code - but on other hand duplicating and maintaining 2 copies of a small util that might not need to be modified in future vs using a popular third-party that seems to handle more scenarios and had dedicated tests etc ..